autoload/OmniSharp.vim

```diff
@@ -527,15 +527,16 @@ function! OmniSharp#StartServerSolution(solutionPath) abort
   let g:OmniSharp_running_slns += [solutionPath]
   let port = exists('b:OmniSharp_port') ? b:OmniSharp_port : g:OmniSharp_port
 
-  let command = [shellescape(g:OmniSharp_server_path, 1),
-               \ '-p', port,
-               \ '-s', shellescape(solutionPath, 1)]
+  let command = join([
+              \ shellescape(g:OmniSharp_server_path, 1),
+              \ '-p', port,
+              \ '-s', shellescape(solutionPath, 1)], ' ')
 
   if g:OmniSharp_server_type !=# 'roslyn'
-    call add(command, OmniSharp#ResolveLocalConfig(solutionPath))
+    command = command . " " . OmniSharp#ResolveLocalConfig(solutionPath)
```

Comment:
I think that it should be `command` instead of `&command`, which would refer to the option with the name `command` and not the variable itself. I do not know vim script well either and am trying to learn it as I go. :)